sender/http.go

```diff
@@ -111,15 +122,29 @@ func (ht *HTTP) init() {
 
 	cp, err := getCertPool(ht.RootCA)
 	if err != nil {
-		httpLog.Print("Failed to initialize root CA pool")
+		httpLog.WithError(err).Error("Failed to initialize root CA pool")
 		return
 	}
 
+	tlsConfig := &tls.Config{
+		InsecureSkipVerify: ht.Insecure,
+		RootCAs:            cp,
+	}
+
+	if ht.Certfile != "" && ht.Keyfile != "" {
```

Comment:
Since both are required if either is specified, we should probably check for this in Verify() to make sure.